addons/missionModules/ACE_Settings.hpp

```diff
@@ -1,128 +1,6 @@
 class ACE_Settings {
+    // TODO: BWC, remove this a while after
     class GVAR(aiSkill_set) {
-        category = QUOTE(PREFIX);
-        displayName = "Global AI Set Skill: Master Enable";
-        typeName = "BOOL";
-        value = 0;
-    };
-    class GVAR(aiSkill_aimingAccuracy_min) {
-        category = QUOTE(PREFIX);
```

Comment:
Would we want `movedToSQF` on all of these?
I think the mission defined ace_settings will cause problems without

Reply:
Yeah I wanted CBA to overwrite the old mission values, but I would have had to change the variables. movedToSQF is needed for sure.